openbao-k8s.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: openbao-k8s
+  version: 1.4.0
+  epoch: 0
+  description: First-class support for OpenBao and Kubernetes.
+  copyright:
+    - license: MPL-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/openbao/openbao-k8s
+      tag: v${{package.version}}
+      expected-commit: 1a8db92701b028327316380db6d634f7d3c5f944
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.23.0
+      replaces: golang.org/x/net=golang.org/x/net@v0.23.0
+
+  - uses: go/bump
+    with:
+      deps: google.golang.org/protobuf@v1.33.0
+      replaces: google.golang.org/protobuf=google.golang.org/protobuf@v1.33.0
```

Comment:
deps and replaces takes more than one args. I think we can combine both of this block.

Reply:
Good point. Thanks, looked at other examples and made it 1 action and simpler.